.vscode/settings.json

```diff
@@ -26,5 +26,8 @@
   "javascript.preferences.importModuleSpecifier": "non-relative",
   "typescript.preferences.importModuleSpecifier": "non-relative",
   "typescript.tsdk": "root/node_modules/typescript/lib",
-  "workbench.editor.labelFormat": "medium"
+  "workbench.editor.labelFormat": "medium",
+  "[typescriptreact]": {
+    "editor.defaultFormatter": "ms-vsliveshare.vsliveshare"
+  }
```

Comment:
@okhaimie-dev this is your config, can you keep it for you only? Pushing this will change all of us settings.

Thanks

Reply:
alright. done

Reply:
> alright. done

Apologize @okhaimie-dev I didn't mean to delete the whole `.vscode` folder, just deleting the new options added on this PR was enough.
The other should remain as they were before please :pray: 

Thanks

Reply:
I will look up ways to achieve this in the am

Reply:
Alright, so the way I can solve this issue is to add `.vscode` folder to the `.gitignore` file after we go back to the initial state. 

I just did that.

Review and let me know if there is anything else. I will get started working on the mobile view.

Reply:
cc: @Markkos89